guides/v2.3/page-builder/index.md

```diff
@@ -3,16 +3,43 @@ layout: full-width
 title: Magento Page Builder Developer Guide
 ---
 
-# What is Page Builder?
+To get started with Page Builder development, visit the [Page Builder Developer Guide]({{ site.baseurl }}/page-builder/docs/). This guide provides overviews, tutorials, and how-to instructions for customizing Page Builder for end-users. 
 
-Page Builder is a Magento extension for creating content by dragging-and-dropping pre-built controls. 
-We call these controls "content types." The available content types are shown in Page Builder's menu:
+The following links provide quick access to our [Page Builder GitHub code examples](https://github.com/magento-devdocs/pagebuilder-examples) and a few key development topics.
 
-![Page Builder content types]({{ site.baseurl }}/page-builder/docs/images/pagebuilder-content-types.png){:width="615px" height="auto"}
+## Page Builder example modules
 
-## Page Builder Developer Guide
+- **[PageBuilderExtensionSlider](https://github.com/magento-devdocs/pagebuilder-examples/tree/master/Example/PageBuilderExtensionSlider)** — Create by [Bruce Denham](https://github.com/bdenham). This module shows you how to add `centerMode` and `centerPadding` settings from the [slick carousel](https://kenwheeler.github.io/slick/) used by the Slider.
```

Comment:
A lot of these say "Create by"; do you mean "Created by" ?

Reply:
Yes, indeed. Thanks!